manifest.json

```diff
@@ -12,19 +12,19 @@
     },
```

Comment:
Signature for `14egjs9cSxt4jnaZTYQE4NemQVXL6qtayqNNpy3VJPSS6AAcaUq`:

`3w0LEilm6ClRRHYEZaKUxY49MCxi0luYF5nu49VJf7outpHG0vB8AFlnfYwoqwlik54te7H16r2vuVMjzsXhAg==`